hackclub.com.yaml

```diff
@@ -1728,11 +1733,18 @@ knit:
 kr:
   ttl: 3600
   type: CNAME
-  value: cname.vercel-dns.com.
```

Comment:
What was previously on this subdomain?

Reply:
Presumably it was reserved for hackclub/global. @juni37 please revert this change